edi/views/edi_exchange_record_views.xml

```diff
@@ -53,12 +53,13 @@
                             <field name="exchange_filename" invisible="1" />
                             <field name="direction" invisible="1" />
                         </group>
-                        <group
-                            name="ack"
-                            attrs="{'invisible': [('ack_needed', '=', False)]}"
-                        >
+                        <!-- TODO: move to a page? -->
```

Comment:
Yes, probably better there, maybe invisible if the type has not an ack_type_id defined

Reply:
I'd leave the TODO there for now. We can change it later.
What about the rest? Approveable? :)